### ISSUE TITLE
PARSE ERROR Could not find stylesheet <all from demo-app>-a11y.css material2-srcs when running mat-switcher -p tsconfig.json

### ISSUE THREAD

**Issue by volei** [angular/material2#7520]
#### Bug, feature request, or proposal:
Maybe I have a *scss/*css misunderstanding... After cloning angular/material2 it npm installed correctly.
After a while I tried 'mat-switcher -p tsconfig.json' to convert to the 'mat-' prefixes instead of 'md-'.
I got the above errors with all missing ...-a11y.css in the demo-app sub-directories.
Is there any naming issue with the stylesheets? They all seem to have *.scss extensions instead. (maybe my lack of knowledge in css/scss).

I don't have this same error when prefix-converting other projects.

#### What is the expected behavior?
find the ...-a11y.css stylesheet files when running the conversion to the md-prefixes to mat-prefixes

#### What is the current behavior?
PARSE ERROR when angular/material-prefix-updater searches for ...-a11y.css stylesheet file

#### What are the steps to reproduce?
cloning angular/material2
run npm install in the cloned directory
npm i -g angular-material-prefix-updater
cd into path/to/project/
mat-switcher -p tsconfig.json

result:
PARSE ERROR ... could not find ...-a11y.css ...

#### What is the use-case or motivation for changing an existing behavior?
convert to mat-prefixes
I used angular/material2 sources for easier access to the examples
Seems like a  *scss/css file naming issue in angular/material2 sources

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 4.4.4 Material 2.0.0.-beta.11 Typescript 2.5.3 Chrome Windows7
Sorry, maybe i first cloned an earlier version of 2.0.0-beta.nn, bit I npm updated to 2.0.0-beta.11. Anyway, after looking into the 2.0.0-beta.11 sources, I see the ...a11y.scss/css descrepancies and still cannot correctly convert to 'mat-'prefixes.

#### Is there anything else we should know?
I am a newbie with css/scss

**Comment by willshowell** [comment]
What is your goal?

**Comment by WaterBleu** [comment]
seems like the same problem I'm having
After running the switcher css failed

**Comment by volei** [comment]
@willshowell, as mentioned, just for convenience: My goal is to run demo-app to try and to copy-paste from material-examples within my IDE.
Maybe I got that wrong and there is another way to access the examples not shown on the webpage other than copy-paste (e.g. material-examples/table-http/table-http-example.*).
But, anyway: As written above, I couln't manage to  'mat-switcher -p tsconfig.json' the repository without ending up with the above error, Without mat-prefixing copy-paste is useless.

**Comment by volei** [comment]
actually was useless, because by now the examples should be mat-prefixed. I had changed the examples and demos during the course, but anyway stumbled over the possible .scss/.css filename issue.

**Comment by willshowell** [comment]
Ah I see. It should have all been switched over already by https://github.com/angular/material2/commit/20a23f16e8ac1ced29172709cc837fa500b6377b. You can see they are already using the `mat-` prefix [here](https://github.com/angular/material2/tree/master/src/material-examples).

**Comment by volei** [comment]
@willshowell: Thanks.

Finally: The filename issue all over in the demo-app and maybe elsewhere seems to be as in this example:

file src/demo-app/chips/chips-demo.ts:
`  styleUrls: ['chips-demo.css']`

But the style-filename themselves are all like
  `chips-demo.scss` (with "s" in extension)

Don't know whether this is intentional or needs correction?!?? **(unsolved)**
At least this confused the mat-converter **(solved)**

**Comment by willshowell** [comment]
During the build process, all the `scss` files are compiled into `css` files in place. Instead of having to rename all the files in `styleUrls` throughout the library, they just prepare for it by naming them what the compiled filename will be.

**Comment by volei** [comment]
@willshowell Thanks very much for clarifying. The `mat-switcher -p tsconfig.json` tool would possibly have to consider this.
**(solved)**
